src/app/home/home.page.scss

```diff
@@ -0,0 +1,30 @@
+mat-card{
+    height: auto;
+    max-width: 350px;
```

Comment:
wherever possible use rem as size unit instead of pixels

Reply:
done